lesson07/assignment/spaceland/src/components/HomePage.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+<div class="hello">
+  <div class="main__h1 d-none d-md-block m-3">
+    <h1 class="text-center display-5">{{ msg }}</h1>
+  </div>
+
+<body>
+  <main class="main">
+
+
+    <div class="main__moon d-flex justify-content-center mt-5 mb-5 pt-5 pb-5">
+      <p class="main__moon__p d-flex align-items-center p-0.5">ORBIT SPACE THIS SEASON!<br/>TICKETS START AT $95</p>
+    </div>
+
+    <div class="d-flex main__end mb-5">
+      <div class="d-flex flex-wrap justify-content-center p-1" data-toggle='tooltip' data-placement='top' title='Check back soon for more details!'>
```

Comment:
This is a good example of something that you could implement using a v-for loop. You'd want to put the image url, title, and other data into an object inside of the data section of your <script>